bin/csv_from_oai.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# OAI client setup
+client = OAI::Client.new(
+  "http://oai.adventistdigitallibrary.org/OAI-script",
+  headers: { from: "jeremy@scientist.com" },
+  parser: 'libxml'
+)
+
+opts = {
+  metadata_prefix: "oai_adl"
+}
+CSV.open('csv_from_oai.csv', 'wb') do |csv|
+  # Write the headers to the CSV file
+  csv << ['aark_id', 'identifier', 'creator',
+          'title', 'resource_type', 'date_created',
+          'language', 'extent', 'source',
+          'geocode', 'place_of_publication', 'publisher',
+          'rights_statement', 'subject', 'related_url',
+          'thumbnail_url', 'work_type']
+  sets = ["adl:thesis"]
```

Comment:
Why are we only querying the "adl:thesis" set?

Reply:
fixed